controllers/migrate.go

```diff
@@ -74,7 +74,7 @@ func migrate(w http.ResponseWriter, r *http.Request) {
 			key, keyErr := logic.RetrievePublicTrafficKey()
 			if keyErr != nil {
 				slog.Error("retrieving traffickey", "error", err)
```

Comment:
this `err` should also be changed. reported by https://github.com/alingse/nilnesserr/actions/runs/14247731195